talktorials/11_online_apis/11a_obtaining_structures.ipynb

```diff
@@ -0,0 +1,2513 @@
+    "    \"issue_comment_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/issues/comments{/number}\",\n",
+    "    \"contents_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/contents/{+path}\",\n",
+    "    \"compare_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/compare/{base}...{head}\",\n",
+    "    \"merges_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/merges\",\n",
+    "    \"archive_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/{archive_format}{/ref}\",\n",
+    "    \"downloads_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/downloads\",\n",
+    "    \"issues_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/issues{/number}\",\n",
+    "    \"pulls_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/pulls{/number}\",\n",
+    "    \"milestones_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/milestones{/number}\",\n",
+    "    \"notifications_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/notifications{?since,all,participating}\",\n",
+    "    \"labels_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/labels{/name}\",\n",
+    "    \"releases_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/releases{/id}\",\n",
+    "    \"deployments_url\": \"https://api.github.com/repos/volkamerlab/TeachOpenCADD/deployments\",\n",
+    "    \"created_at\": \"2018-11-09T13:15:15Z\",\n",
+    "    \"updated_at\": \"2019-07-18T02:36:48Z\",\n",
+    "    \"pushed_at\": \"2019-05-03T15:02:03Z\",\n",
+    "    \"git_url\": \"git://github.com/volkamerlab/TeachOpenCADD.git\",\n",
+    "    \"ssh_url\": \"git@github.com:volkamerlab/TeachOpenCADD.git\",\n",
+    "    \"clone_url\": \"https://github.com/volkamerlab/TeachOpenCADD.git\",\n",
+    "    \"svn_url\": \"https://github.com/volkamerlab/TeachOpenCADD\",\n",
+    "    \"homepage\": null,\n",
+    "    \"size\": 28121,\n",
+    "    \"stargazers_count\": 39,\n",
+    "    \"watchers_count\": 39,\n",
+    "    \"language\": \"Jupyter Notebook\",\n",
+    "    \"has_issues\": true,\n",
+    "    \"has_projects\": true,\n",
+    "    \"has_downloads\": true,\n",
+    "    \"has_wiki\": true,\n",
+    "    \"has_pages\": false,\n",
+    "    \"forks_count\": 13,\n",
+    "    \"mirror_url\": null,\n",
+    "    \"archived\": false,\n",
+    "    \"disabled\": false,\n",
+    "    \"open_issues_count\": 0,\n",
+    "    \"license\": {\n",
+    "      \"key\": \"cc-by-4.0\",\n",
+    "      \"name\": \"Creative Commons Attribution 4.0 International\",\n",
+    "      \"spdx_id\": \"CC-BY-4.0\",\n",
+    "      \"url\": \"https://api.github.com/licenses/cc-by-4.0\",\n",
+    "      \"node_id\": \"MDc6TGljZW5zZTI1\"\n",
+    "    },\n",
+    "    \"forks\": 13,\n",
+    "    \"open_issues\": 0,\n",
+    "    \"watchers\": 39,\n",
+    "    \"default_branch\": \"master\"\n",
+    "  }\n",
+    "]\n",
```

Comment:
This chunk between "Returns" and ", which happens to be ..." does not show up in my Jupyter Notebook.

Reply:
You mean the GitHub preview? In your local preview, you need to click on "> Returns" to reveal the hidden block.

Reply:
Oh, that's nice! 

For me, it was not intuitive to click on the word "Returns". 
Maybe you could add a "click here" and avoid the sentence split (single word "Returns" + new line starting with a comma) by writing:

> Requesting the URL via API returns a JSON-formatted dictionary (click here)!
> 
> This can easily be parsed...